_episodes/15-transferring-files.md

```diff
@@ -100,18 +100,23 @@ the transfer.
 
 The most common archiving command you will use on (Linux) HPC cluster is `tar`. `tar` can be used
 to combine files into a single archive file and, optionally, compress. For example, to combine
-all files within a folder called `output_data` into an archive file called `output_data.tar` we
-would use:
+all files of the pattern `output_data/output00001.out`, `output_data/output00002.out`, ... within a folder called `output_data` into an archive file called `output_data.tar` we would use:
 
 ```
-tar -cvf output_data.tar output_data/
+{{ site.local_prompt }} tar -cvf output_data.tar output_data/
 ```
 {: .bash}
 
-We also use the `tar` command to extract the files from the archive once we have transferred it:
+The options we used for `tar` are:
+
+- `-c` - Create new archive
+- `-v` - Verbose (print what you are doing!)
+- `-f mydata.tar` - Create the archive in file *output_data.tar*
+
+The tar command allows users to concatenate flags. Instead of typing `tar -c -v -f`, we can do `tar -cvf`. We can also use the `tar` command to extract the files from the archive once we have transferred it:
```

Comment:
```suggestion
The tar command allows users to concatenate flags. Instead of typing `tar -c -v -f`, we can use `tar -cvf`. We can also use the `tar` command to extract the files from the archive once we have transferred it:
```

Reply:
fixed in 0f80685